README.md

```diff
@@ -229,6 +229,27 @@ The following command line options only apply when measuring Gradle builds:
 - `--no-studio-sandbox`: Do not use the Android Studio sandbox but use the default Android Studio folders for the Android Studio data.
 - `--no-diffs`: Do not generate differential flame graphs.
 
+## JVM requirements and options
```

Comment:
❓ Could we also mention that Gradle 6+ is tested and earlier versions are not anymore? And possibly thatuser should use some old gradle-profiler to test older versions